app/src/main/java/com/duckduckgo/app/browser/omnibar/OmnibarLayout.kt

```diff
@@ -607,30 +581,6 @@ class OmnibarLayout @JvmOverloads constructor(
         viewModel.onExternalStateChange(stateChange)
     }
 
-    override fun setExpanded(expanded: Boolean) {
```

Comment:
this now belongs to the `ScrollingOmnibarLayout`, which is the one that hides when the user scrolls.